cli/hzn.go

```diff
@@ -72,7 +72,7 @@ Environment Variables:
 
 	exchangeCmd := app.Command("exchange", "List and manage Horizon Exchange resources.")
```

Comment:
You also need to modify the text describing the HZN_EXCHANGE_USER_AUTH env var.

Reply:
updated HZN_EXCHANGE_USER-AUTH environment variable, ran all tests successfully